### ISSUE TITLE
Error on `vagrant up`

### ISSUE THREAD

**Issue by andrioni** [snowplow/sql-runner#28]
```
==> default: =======================================
==> default: CLONING ANSIBLE AND PLAYBOOKS WITH PERU
==> default: ---------------------------------------
==> default: Traceback (most recent call last):
==> default:   File "/usr/local/bin/peru", line 9, in <module>
==> default:     load_entry_point('peru==0.2.4', 'console_scripts', 'peru')()
==> default:   File "/usr/local/lib/python3.4/dist-packages/peru/main.py", line 314, in main
==> default:     scope, imports = parser.parse_file(runtime.peru_file)
==> default:   File "/usr/local/lib/python3.4/dist-packages/peru/parser.py", line 22, in parse_file
==> default:     return parse_string(f.read(), name_prefix)
==> default:   File "/usr/local/lib/python3.4/dist-packages/peru/parser.py", line 32, in parse_string
==> default:     return _parse_toplevel(blob, name_prefix)
==> default:   File "/usr/local/lib/python3.4/dist-packages/peru/parser.py", line 36, in _parse_toplevel
==> default:     modules = _extract_modules(blob, name_prefix)
==> default:   File "/usr/local/lib/python3.4/dist-packages/peru/parser.py", line 100, in _extract_modules
==> default:     yaml_name)
==> default:   File "/usr/local/lib/python3.4/dist-packages/peru/parser.py", line 107, in _build_module
==> default:     default_rule = _extract_default_rule(blob)
==> default:   File "/usr/local/lib/python3.4/dist-packages/peru/parser.py", line 84, in _extract_default_rule
==> default:     return _extract_rule("<default>", blob)
==> default:   File "/usr/local/lib/python3.4/dist-packages/peru/parser.py", line 68, in _extract_rule
==> default:     untar/unzip a curl module, use the "unpack" field.'''))
==> default: peru.parser.ParserError: The "build" field is no longer supported. If you need to
==> default: untar/unzip a curl module, use the "unpack" field.
```

I'm using Vagrant 1.7.2, if it helps.

**Comment by andrioni** [comment]
Oh, and this is on the master branch (0.1.0). I'm testing it right now on 0.2.0.

**Comment by alexanderdean** [comment]
Thanks for raising, will be fixed by #25
